src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,73 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+import { FilterType } from '../../utils/FilterType';
+
+interface Props {
+  count: number;
+  filterBy: FilterType;
+  setFilterBy: (filterBy: FilterType) => void;
+  isSomeTodoCompleted: boolean;
+  handleClear: () => void;
+}
+
+export const Footer: FC<Props> = (
+  {
+    filterBy,
+    setFilterBy,
+    isSomeTodoCompleted,
+    handleClear,
+    count,
+  },
+) => {
+  return (
+    <footer className="footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${count} items left`}
```

Comment:
Add condition for words `items` or `item` when you have only 1 todo
```suggestion
        {`${count} items left`}
```